package.json

```diff
@@ -27,27 +27,26 @@
   },
   "dependencies": {
     "@sentry/node": "^5.27.1",
-    "bluebird": "^3.1.1",
```

Comment:
A number of these packages have had minor or major version bumps, but I checked the various changelogs and it all looks fine. The tests complete and I did a trial run with my local box.